charts/internal/ff-proxy/values.yaml

```diff
@@ -241,7 +241,7 @@ readReplica:
 
 #common configs which other services use.
 config:
-  ENV: SMP
+  ENV: qa
```

Comment:
Was this intentional, or should we default to SMP?

Reply:
It was intentional to fix that issue with the env name for the profiler in QA.